octavia-cli/integration_tests/configurations/connections/poke_to_pg/configuration.yaml

```diff
@@ -1,38 +1,35 @@
-# Configuration for connection poke_to_g
+# Configuration for connection poke_to_pg
 definition_type: connection
-resource_name: poke_to_g
+resource_name: poke_to_pg
 source_id: TO_UPDATE_FROM_TEST
 destination_id: TO_UPDATE_FROM_TEST
 
 # EDIT THE CONFIGURATION BELOW!
 configuration:
-  sourceId: TO_UPDATE_FROM_TEST # REQUIRED | string
-  destinationId: TO_UPDATE_FROM_TEST # REQUIRED | string
   status: active # REQUIRED | string | Allowed values: active, inactive, deprecated
-  name: poke_to_g # OPTIONAL | string | Optional name of the connection
-  namespaceDefinition: source # OPTIONAL | string | Allowed values: source, destination, customformat
-  namespaceFormat: "${SOURCE_NAMESPACE}" # OPTIONAL | string | Used when namespaceDefinition is 'customformat'. If blank then behaves like namespaceDefinition = 'destination'. If "${SOURCE_NAMESPACE}" then behaves like namespaceDefinition = 'source'.
+  namespace_definition: source # OPTIONAL | string | Allowed values: source, destination, customformat
+  namespace_format: "${SOURCE_NAMESPACE}" # OPTIONAL | string | Used when namespaceDefinition is 'customformat'. If blank then behaves like namespaceDefinition = 'destination'. If "${SOURCE_NAMESPACE}" then behaves like namespaceDefinition = 'source'.
   prefix: "" # REQUIRED | Prefix that will be prepended to the name of each stream when it is written to the destination
-  resourceRequirements: # OPTIONAL | object | Resource requirements to run workers (blank for unbounded allocations)
+  resource_requirements: # OPTIONAL | object | Resource requirements to run workers (blank for unbounded allocations)
```

Comment:
Try to catch api exception or validate existing configs to provide a understandable error message to the user directing them to switch to snake_case, or re-generate the files.